Makefile

```diff
@@ -1,6 +1,7 @@
 test: test_lint test_unit test_integration
 
 test_lint:
+	git config --global --add safe.directory /srv/cartography
```

Comment:
As far as what I can understand is happening in this container is that the last line of the dockerfile isn't translating into the cartography user owning the directory. Git picks this up and throws an error where it detects dubious ownership in the repository. See #1269. This then tells git that `/srv/cartography` is a safe directory and any user can run git commands on it. 

This is only a dev container so there shouldn't be any security issues.

Reply:
Some devs might use a local python install instead of Docker for testing so the path to cartography on their machines won't be /srv/cartography.

Just to test, I ran `git config --global --add safe.directory /srv/cartography` on my machine and it doesn't throw an error even though the path doesn't exist, so even though it's not the cleanest way to do it, it's harmless.

Ideally we'd solve the permission ownership problem though.